examples/fullscreen.rs

```diff
@@ -118,14 +83,39 @@ fn prompt_for_monitor(event_loop: &EventLoop<()>) -> MonitorHandle {
     }
 
     print!("Please write the number of the monitor to use: ");
-    io::stdout().flush().unwrap();
+    stdout().flush().unwrap();
 
     let mut num = String::new();
-    io::stdin().read_line(&mut num).unwrap();
+    stdin().read_line(&mut num).unwrap();
     let num = num.trim().parse().ok().expect("Please enter a number");
-    let monitor = event_loop.available_monitors().nth(num).expect("Please enter a valid ID");
+    let monitor = event_loop
+        .available_monitors()
+        .nth(num)
+        .expect("Please enter a valid ID");
 
     println!("Using {:?}", monitor.name());
 
     monitor
 }
+
+fn prompt_for_video_mode(monitor: &MonitorHandle) -> VideoMode {
+    // Video modes are returned in a random order, so we must store them in
+    // order to be able to later pick the nth video mode reliably
+    let mut video_modes: Vec<_> = monitor.video_modes().collect();
```

Comment:
For the sake of consistency when running this example, could the video modes get sorted here?